packages/vscode-extension/test/index.integration.ts

```diff
@@ -15,7 +15,7 @@ export async function run(): Promise<void> {
     reporterOptions: {
       reporterEnabled: "spec, mocha-junit-reporter",
       mochaJunitReporterReporterOptions: {
-        mochaFile: path.resolve(__dirname, "..", "..", "test-results.nofolder.xml"),
+        mochaFile: path.resolve(__dirname, "..", "..", "test-results.integration.xml"),
```

Comment:
do we have integration test now?

Reply:
Currently there's one. The original structure is <nofolder> = <unit test no folder> + <integration test no folder>, I separated according to unit test and integration test now. Should I clear the integration part in this PR?